charts/README.md

```diff
@@ -157,6 +157,7 @@ $ helm install karmada-scheduler-estimator -n karmada-system ./charts
 | `etcd.internal.pvc.storageClass`          | StorageClass of the etcd, takes effect when `etcd.internal.storageType` is "pvc"                                                                                                                                                              | `""`                                                                                                                                                       |
 | `etcd.internal.pvc.size`                  | Storage size of the etcd, takes effect when `etcd.internal.storageType` is "pvc"                                                                                                                                                              | `""`                                                                                                                                                       |
 | `etcd.internal.resources`                 | Resource quota of the etcd                                                                                                                                                                                                                    | `{}`                                                                                                                                                       |
+| `etcd.strategy`                           | Strategy of the etcd                                                                                                                                                                                                                          | `{"type": "RollingUpdate", "rollingUpdate": {"maxUnavailable": "0", "maxSurge": "50%"} }`                                                                  |
```

Comment:
etcd doesn't appear to have a rollingupdate policy configured. I guess the default value is not {"maxUnavailable": "0", "maxSurge": "50%"}. BTW, others should be checked that doc and yaml files keep the same.